gradle.properties

```diff
@@ -1,6 +1,5 @@
 projectVersion=4.4.0-SNAPSHOT
 projectGroup=io.micronaut.kafka
-micronautSharedSettingVersion=5.3.10
```

Comment:
I have moved this directly to settings to align to the [project template](https://github.com/micronaut-projects/micronaut-project-template/pull/246)